verifyFiles.py

```diff
@@ -279,24 +328,32 @@ def main():
                                              /home/$(USER)/CentOS6.FSL5.0.6
                                              /home/$(USER)/CentOS7.FSL5.0.6
                                              Each directory will contain subject folders like 100307,100308 etc'''))
-        parser.add_argument("-c", "--checksumFile",action="store_true",help="Reads checksum from files. Doesn't compute checksums locally")
+        parser.add_argument("-c", "--checksumFile",help="Reads checksum from files. Doesn't compute checksums locally")
```

Comment:
Instead of passing a boolean value here, pass the path of the checksums file relatively to the subject folder.